WORKSPACE

```diff
@@ -86,3 +86,18 @@ tf_workspace1()
 load("@//tensorflow:workspace0.bzl", "tf_workspace0")
 
 tf_workspace0()
+
+# Hedron's Compile Commands Extractor for Bazel
```

Comment:
Why is this needed?

Reply:
This allows us to generate file `compile_commands.json` in the root directly by running command `bazelisk run tensorflow/compiler/mlir:tf_opt_compile_commands -- --config=avx2_linux --config=dbg`. Useful for LSP.

I'm aware this change is unrelated with the proposed feature. If you'd like I can submit this as a separate pull request, or remove it altogether if Tensorflow already has a way to integrate with LSP that I'm not aware of. Let me know how you'd like me to proceed.